HLTrigger/Tools/bin/hltDiff.cc

```diff
@@ -747,49 +747,63 @@ class JsonOutputProducer {
         // Writing the output to a JSON file
         std::string output_name = output_filename_base(run) += ".json";
         out_file.open(output_name, std::ofstream::out);
+        if (out_file.good()) {
+          out_file << '{';  // line open
+          out_file << configuration.serialise(1) << ',';
+          out_file << vars.serialise(1) << ',';
+          // writing block for each event
+          out_file << indent(1) << key("events") << '[';  // line open
+          for (auto it = v_events.begin(); it != v_events.end(); ++it) {
+            out_file << (*it).serialise(2);
+            if (it != --v_events.end())
+              out_file << ',';
+          }
+          out_file << indent(1) << ']';  // line close
+          out_file << indent(0) << "}";  // line close
+          out_file.close();
+          // Adding file name to the list of created files
+          filesCreated.insert(output_name);
+        } else
+          filesNotCreated.insert(output_name);
+      }
+    } else {
+      // Creating a single file containing with only configuration part
+      std::string output_name = output_filename_base(0) += ".json";
+      out_file.open(output_name, std::ofstream::out);
+      if (out_file.good()) {
         out_file << '{';  // line open
         out_file << configuration.serialise(1) << ',';
         out_file << vars.serialise(1) << ',';
         // writing block for each event
         out_file << indent(1) << key("events") << '[';  // line open
-        for (auto it = v_events.begin(); it != v_events.end(); ++it) {
-          out_file << (*it).serialise(2);
-          if (it != --v_events.end())
-            out_file << ',';
-        }
+        // for (std::vector<JsonEvent>::const_iterator it = v_events.begin(); it != v_events.end(); ++it) {
+        //   out_file << (*it).serialise(2);
+        //   if (it != --v_events.end()) out_file << ',';
+        // }
```

Comment:
@missirol should these commented-out lines be uncommented, kept, or removed ?

Reply:
Hi @fwyzard , not sure. I didn't touch those (despite what the diff suggests..); I think they are just [these lines](https://github.com/cms-sw/cmssw/pull/33174/files#diff-7a488f2c7fdc3b0f6ccdaab9428d9a61632af34963fc1b8dfbf17bd77d61d220L775); indentation changes make it look like they were changed, but they weren't. From the looks of it, they could be probably removed.